xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -44,8 +44,8 @@
 #define ADDON_GLOBAL_VERSION_GENERAL_XML_ID           "kodi.binary.global.general"
 #define ADDON_GLOBAL_VERSION_GENERAL_DEPENDS          "General.h"
 
-#define ADDON_GLOBAL_VERSION_GUI                      "5.12.1"
-#define ADDON_GLOBAL_VERSION_GUI_MIN                  "5.10.0"
+#define ADDON_GLOBAL_VERSION_GUI                      "5.13.0"
+#define ADDON_GLOBAL_VERSION_GUI_MIN                  "5.13.0"
```

Comment:
Here it must be updated again, have added with https://github.com/xbmc/xbmc/commit/983051bbebbe6ab7cd02b0f0c87dbacf2f97b1cb the version on another part.
SORRY!

Reply:
First come first serve, I'd say. If your PR goes in after this one you are the one who has to bump to 5.14.0 ;-)

Reply:
PR is already merged. It doesn't conflict because it's the exact same change.

Reply:
Just noticed your incompatible breaking change PR was already merged?! Without increasing version number of affected binary addons? Do we need to have this discussion with every incompatible addon API change you introduce @AlwinEsch  ? When bumping API version affected addons must get an addon version bump!

Reply:
> PR is already merged. It doesn't conflict because it's the exact same change.

 Then we actually need to bump to 5.14.0 with this PR.

Reply:
No there is no need for add-on version bump, because they are not recompiled automatically. They will just not be installable / compatible with the current Kodi version.

Reply:
Agree with you. Just find it in an alpha version, do not overdo it, otherwise you will not finish with version updates.

Reply:
>  because they are not recompiled automatically. 

 Not quite true. @MilhouseVH for instance rebuilds addons every night, IIRC. To be on the safe side, we need to bump addon-version.

Reply:
> Just find it in an alpha version, do not overdo it

 I tend to disagree.

 Are you the one willing to take care of the support requests / issues our alpha users report wrt crashing/not installing/crazy behaving binary addons? I'm not.

Reply:
I'll bump it to .14 and do the necessary version bump PR's to every single binary addon that uses `kodi.binary.global.gui`. Quite a bit of "monkey" work ahead when I just wanted to break the calls from python :)